### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by WaleedAshraf**
Links broke probably in this PR https://github.com/openjs-foundation/cross-project-council/pull/841

Twitter status links keeps failing randomly. I have added them to skip.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/regular-member-pull-request.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ about: You'd like to participate in the Cross Project Council's work as a Regula
 We are thrilled that you'd like to participate in our work in a meaningful way as a Regular Member of the Cross Project Council.
 
 Requirements and rules around becoming a Regular Member can be found in our Governance:
-https://github.com/openjs-foundation/cross-project-council/blob/HEAD/GOVERNANCE.md#approving-and-onboarding-regular-members
+https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#approving-and-onboarding-regular-members
```
Comment (ljharb) [inline .github/PULL_REQUEST_TEMPLATE/regular-member-pull-request.md:10]:
```suggestion
https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/GOVERNANCE.md#approving-and-onboarding-regular-members
```

File: .github/PULL_REQUEST_TEMPLATE/regular-member-pull-request.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ about: You'd like to participate in the Cross Project Council's work as a Regula
...
 
 Please fill out information below.
 
 Thanks!
 -->
 
-Please describe your recent participation in the work of the foundation or its member projects as described in the [CPC Governance requirements for becoming a Regular Member](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/GOVERNANCE.md#approving-and-onboarding-regular-members)
+Please describe your recent participation in the work of the foundation or its member projects as described in the [CPC Governance requirements for becoming a Regular Member](https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#approving-and-onboarding-regular-members)
```
Comment (ljharb) [inline .github/PULL_REQUEST_TEMPLATE/regular-member-pull-request.md:17]:
```suggestion
Please describe your recent participation in the work of the foundation or its member projects as described in the [CPC Governance requirements for becoming a Regular Member](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/GOVERNANCE.md#approving-and-onboarding-regular-members)
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -32,14 +32,14 @@ For reporting issues in spaces managed by the OpenJS Foundation, for example, re
 
 ## Escalate an issue
 
-The OpenJS Foundation maintains a [Code of Conduct Panel (CoCP)](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md#code-of-conduct-panel). This is a foundation-wide team established to manage escalation when a reporter believes that a report to a member project or the CPC has not been properly handled.
+The OpenJS Foundation maintains a [Code of Conduct Panel (CoCP)](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md#code-of-conduct-panel). This is a foundation-wide team established to manage escalation when a reporter believes that a report to a member project or the CPC has not been properly handled.
```
Comment (ljharb) [inline CODE_OF_CONDUCT.md:35]:
```suggestion
The OpenJS Foundation maintains a [Code of Conduct Panel (CoCP)](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md#code-of-conduct-panel). This is a foundation-wide team established to manage escalation when a reporter believes that a report to a member project or the CPC has not been properly handled.
```

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -32,14 +32,14 @@ For reporting issues in spaces managed by the OpenJS Foundation, for example, re
...
 
 In order to escalate to the CoCP, email <coc-escalation@lists.openjsf.org>.
 
 ## More Info
 
 For more information, refer to the full
-[Code of Conduct governance document](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md).
+[Code of Conduct governance document](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md).
```
Comment (ljharb) [inline CODE_OF_CONDUCT.md:42]:
```suggestion
[Code of Conduct governance document](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md).
```

File: CPC-CHARTER.md [hunk 5]
```diff
@@ -334,5 +334,5 @@ policies.
 [Consensus Seeking]: http://en.wikipedia.org/wiki/Consensus-seeking_decision-making
 [Condorcet]: http://en.wikipedia.org/wiki/Condorcet_method
 [Single Transferable Vote]: http://en.wikipedia.org/wiki/Single_transferable_vote
-[Active OpenJS Collaborator]: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/GOVERNANCE.md#definition-of-an-active-openjs-collaborator
+[Active OpenJS Collaborator]: https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#definition-of-an-active-openjs-collaborator
```
Comment (ljharb) [inline CPC-CHARTER.md:337]:
```suggestion
[Active OpenJS Collaborator]: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/GOVERNANCE.md#definition-of-an-active-openjs-collaborator
```

File: PROJECT_PROGRESSION.md [hunk 6]
```diff
@@ -103,11 +103,11 @@ This is an informational checklist to help projects onboard into the OpenJS Foun
 - [ ] Update project CoC reporting methods to include OpenJS Foundation escalation path
 - [ ] List official domains that the project commits to transfer to the OpenJS Foundation following graduation
 - [ ] Identify and document other core project infrastructure
-- [ ] Adopt either the [Contributor License Agreement (CLA) or the Developer Certificate of Origin (DCO)](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/IP_POLICY_GUIDANCE.md#4-adopting-the-dco-or-a-cla)
+- [ ] Adopt either the [Contributor License Agreement (CLA) or the Developer Certificate of Origin (DCO)](https://github.com/openjs-foundation/cross-project-council/blob/main/governance/IP_POLICY_GUIDANCE.md#4-adopting-the-dco-or-a-cla)
```
Comment (ljharb) [inline PROJECT_PROGRESSION.md:106]:
```suggestion
- [ ] Adopt either the [Contributor License Agreement (CLA) or the Developer Certificate of Origin (DCO)](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/IP_POLICY_GUIDANCE.md#4-adopting-the-dco-or-a-cla)
```

File: PROJECT_PROGRESSION.md [hunk 7]
```diff
@@ -103,11 +103,11 @@ This is an informational checklist to help projects onboard into the OpenJS Foun
...
 - [ ] Add or Update Governance.md document (required for Impact stage)
 - [ ] Confirm required files in place (CODE_OF_CONDUCT.md, LICENSE.md)
 - [ ] Publish Project Charter on website or GitHub (see [charter template](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/PROJECT_CHARTER_TEMPLATE.md))
-- [ ] Update [legal copyright notice](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/IP_POLICY_GUIDANCE.md#2-copyright-notices) on GitHub
+- [ ] Update [legal copyright notice](https://github.com/openjs-foundation/cross-project-council/blob/main/governance/IP_POLICY_GUIDANCE.md#2-copyright-notices) on GitHub
```
Comment (ljharb) [inline PROJECT_PROGRESSION.md:110]:
```suggestion
- [ ] Update [legal copyright notice](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/IP_POLICY_GUIDANCE.md#2-copyright-notices) on GitHub
```

File: conduct/HANDLING_CODE_OF_CONDUCT_REPORTS.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # Handling reports and escalation
 This document describes the proposed process for handling reports and escalation, along with relevant roles in the OpenJS Foundation.
 
-This process covers two types of reports based on the [Foundation's Code of Conduct Requirements](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md):
+This process covers two types of reports based on the [Foundation's Code of Conduct Requirements](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md):
```
Comment (ljharb) [inline conduct/HANDLING_CODE_OF_CONDUCT_REPORTS.md:4]:
```suggestion
This process covers two types of reports based on the [Foundation's Code of Conduct Requirements](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md):
```

File: conduct/HANDLING_CODE_OF_CONDUCT_REPORTS.md [hunk 9]
```diff
@@ -51,7 +51,7 @@ All members of the CPC are subscribed to the `report@openjsf.org` mailing list.
 
 Note: We understand that the reporter and the target can be separate persons. In this case the CPC will communicate with the reporter unless the target gives permission for the CPC to communicate with them.
 
-All members of the [Code of Conduct Panel (CoCP)](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md#code-of-conduct-panel)
+All members of the [Code of Conduct Panel (CoCP)](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md#code-of-conduct-panel)
```
Comment (ljharb) [inline conduct/HANDLING_CODE_OF_CONDUCT_REPORTS.md:54]:
```suggestion
All members of the [Code of Conduct Panel (CoCP)](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md#code-of-conduct-panel)
```

File: governance/WORKING_GROUPS.md [hunk 10]
```diff
@@ -139,7 +139,7 @@ The purpose of the Code of Conduct Working Group is to define and maintain a set
 Responsibilities include:
 
 * Define and maintain overarching Code of Conduct and moderation processes, documentation and resources as they pertain to the Foundation and the Cross Project Council.
-* Provide guidance/resources to implement the minimum Code of Conduct and moderation requirements for projects as part of Foundation membership as defined in the [Foundation Code of Conduct Requirements](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md).
+* Provide guidance/resources to implement the minimum Code of Conduct and moderation requirements for projects as part of Foundation membership as defined in the [Foundation Code of Conduct Requirements](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md).
```
Comment (ljharb) [inline governance/WORKING_GROUPS.md:142]:
```suggestion
* Provide guidance/resources to implement the minimum Code of Conduct and moderation requirements for projects as part of Foundation membership as defined in the [Foundation Code of Conduct Requirements](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md).
```

File: governance/transfer-repo-into-org.md [hunk 11]
```diff
@@ -41,6 +41,6 @@ See [GitHub's documentation on transferring repos][] on how to perform the neces
 
 
 [GitHub's documentation on transferring repos]: https://help.github.com/articles/about-repository-transfers/
-[OpenJS Foundation GitHub Organization Management Policy]: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/GITHUB_ORG_MANGEMENT_POLICY.md
+[OpenJS Foundation GitHub Organization Management Policy]: https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GITHUB_ORG_MANAGEMENT_POLICY.md
```
Comment (ljharb) [inline governance/transfer-repo-into-org.md:44]:
```suggestion
[OpenJS Foundation GitHub Organization Management Policy]: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/GITHUB_ORG_MANAGEMENT_POLICY.md
```

File: meetings/2019/2019-04-01.md [hunk 12]
```diff
@@ -29,7 +29,7 @@ Extracted from **bootstrap-agenda** labelled issues and pull requests from the *
 
 * [BW] Infrastructure: Mailing lists  (5 min) [#134](https://github.com/openjs-foundation/bootstrap/issues/134)
   * What do we want to create comms-wise, what do we specifically not want to create?
-  * From: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md we’d need coc-escalation@xxx.org and report@xxxx.org
+  * From: https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md we’d need coc-escalation@xxx.org and report@xxxx.org
```
Comment (ljharb) [inline meetings/2019/2019-04-01.md:32]:
```suggestion
  * From: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md we’d need coc-escalation@xxx.org and report@xxxx.org
```

File: meetings/2020/2020-09-29.md [hunk 13]
```diff
@@ -45,7 +45,7 @@
   * No license in the repo right now?
   * Possibly MIT but unclear.
   * Can we relicense CC?
-  * MIT is OK for documentation (see https://github.com/openjs-foundation/cross-project-council/blob/HEAD/IP_POLICY_GUIDANCE.md#1-licensing
+  * MIT is OK for documentation (see https://github.com/openjs-foundation/cross-project-council/blob/main/governance/IP_POLICY_GUIDANCE.md#1-licensing
```
Comment (ljharb) [inline meetings/2020/2020-09-29.md:48]:
```suggestion
  * MIT is OK for documentation (see https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/IP_POLICY_GUIDANCE.md#1-licensing
```

File: meetings/2020/2020-12-15.md [hunk 14]
```diff
@@ -40,7 +40,7 @@
 * Prepare to open nominations for 2nd Board seat [#687](https://github.com/openjs-foundation/cross-project-council/issues/687)
  * one seat is up for election early in new year (CPC representative)
  * open call for nominees the first meeting of 2021
- * This describes the expectations - https://github.com/openjs-foundation/cross-project-council/blob/HEAD/COMMUNITY_BOARD_SEAT_EXPECTATIONS.md
+ * This describes the expectations - https://github.com/openjs-foundation/cross-project-council/blob/main/governance/COMMUNITY_BOARD_SEAT_EXPECTATIONS.md
```
Comment (ljharb) [inline meetings/2020/2020-12-15.md:43]:
```suggestion
 * This describes the expectations - https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/COMMUNITY_BOARD_SEAT_EXPECTATIONS.md
```

File: meetings/2021/2021-05-25.md [hunk 15]
```diff
@@ -28,7 +28,7 @@
 ### openjs-foundation/cross-project-council
 
 * approve Node.js TSC Charter revisions [#762](https://github.com/openjs-foundation/cross-project-council/issues/762)
-  * Just waiting now for 14 days as required by -https://github.com/openjs-foundation/cross-project-council/blob/HEAD/GOVERNANCE.md#approving-project-charters
+  * Just waiting now for 14 days as required by -https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#approving-project-charters
```
Comment (ljharb) [inline meetings/2021/2021-05-25.md:31]:
```suggestion
  * Just waiting now for 14 days as required by -https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/GOVERNANCE.md#approving-project-charters
```

File: meetings/2021/2021-07-20.md [hunk 16]
```diff
@@ -48,7 +48,7 @@ Book club call is upcoming
 * Clarify
 [#778](https://github.com/openjs-foundation/cross-project-council/issues/778)
   * Is having a DCO or CLA required? CPC says yes; Robin or Michael will find the source that sets the requirement. 
-  * Yes as per https://github.com/openjs-foundation/cross-project-council/blob/HEAD/IP_POLICY_GUIDANCE.md
+  * Yes as per https://github.com/openjs-foundation/cross-project-council/blob/main/governance/IP_POLICY_GUIDANCE.md
```
Comment (ljharb) [inline meetings/2021/2021-07-20.md:51]:
```suggestion
  * Yes as per https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/IP_POLICY_GUIDANCE.md
```

File: meetings/2021/2021-08-03.md [hunk 17]
```diff
@@ -49,7 +49,7 @@
 * Clarify whether it is a requirement to use either CLA or DCO at OpenJS Foundation [#778](https://github.com/openjs-foundation/cross-project-council/issues/778)
   * Remaining question is if we should update the IP policy to clarify you have to
     use one or the other?
-  * comment with approach which is: The final discussion was that we'll keep this open. If/when there is some other update to the IP policy then we'll bring up clarifying this point. In the mean time, the guidance doc is pretty clear: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/IP_POLICY_GUIDANCE.md#4-adopting-the-dco-or-a-cla
+  * comment with approach which is: The final discussion was that we'll keep this open. If/when there is some other update to the IP policy then we'll bring up clarifying this point. In the mean time, the guidance doc is pretty clear: https://github.com/openjs-foundation/cross-project-council/blob/main/governance/IP_POLICY_GUIDANCE.md#4-adopting-the-dco-or-a-cla
```
Comment (ljharb) [inline meetings/2021/2021-08-03.md:52]:
```suggestion
  * comment with approach which is: The final discussion was that we'll keep this open. If/when there is some other update to the IP policy then we'll bring up clarifying this point. In the mean time, the guidance doc is pretty clear: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/IP_POLICY_GUIDANCE.md#4-adopting-the-dco-or-a-cla
```

File: proposals/incubating/CHARTER_REVIEW/charter_review.md [hunk 18]
```diff
@@ -44,4 +44,4 @@ CPC members should add themselves as reviewers to the associated charter issue.
 
 Project Charters should be reviewed and approved within two weeks of submission. An exception may be made if the charter is submitted during a holiday period (such as the end of the year) or in the event either the project or CPC has requested legal review. 
 
-Charters are approved when they meet the requirements outlined in the 'Approving Charters' section of [GOVERNANCE.md](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/GOVERNANCE.md#approving-project-charters).
+Charters are approved when they meet the requirements outlined in the 'Approving Charters' section of [GOVERNANCE.md](https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#approving-project-charters).
```
Comment (ljharb) [inline proposals/incubating/CHARTER_REVIEW/charter_review.md:47]:
```suggestion
Charters are approved when they meet the requirements outlined in the 'Approving Charters' section of [GOVERNANCE.md](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/GOVERNANCE.md#approving-project-charters).
```

**Review by ljharb — COMMENTED** [review]
Thanks!

**Review by ljharb — APPROVED** [review]
Fast track?

**Review by joesepi — APPROVED** [review]
Thanks!

**Comment by joesepi** [comment]
Fast tracking upvotes and approvals received. Merging this puppy.

**Pull request merged**